driver_cpl/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -4,6 +4,275 @@
 
 <namelist_defaults>
 
+<!-- =========================================  -->
```

Comment:
@mvertens , I don't see this file: driver_cpl/cime_config/namelist_defaults_drv.xml. Did you mean driver_cpl/cime_config/namelist_definition_drv.xml?